frontend/src/locales/scripts/en.json5

```
@@ -646,7 +646,7 @@
        * Interpolated into browsePage.searchForm.placeholder:
        * _{type}_ part of "Search all {type}"
        */
-      audio: "audio",
+      audio: "audio track",
```

Please use the plural form.
```suggestion
      audio: "audio tracks",
```